packages/block-library/src/social-link/variations.js

```diff
@@ -341,16 +341,22 @@ const variations = [
 ];
 
 /**
- * Add `isActive` function to all `social link` variations, if not defined.
- * `isActive` function is used to find a variation match from a created
- *  Block by providing its attributes.
+ * Add `isActive` and `example` properties to all Social Link variations, if not defined.
  */
 variations.forEach( ( variation ) => {
-	if ( variation.isActive ) {
-		return;
+	if ( ! variation.isActive ) {
+		variation.isActive = ( blockAttributes, variationAttributes ) =>
+			blockAttributes.service === variationAttributes.service;
+	}
+
+	if ( ! variation.example ) {
+		variation.example = {
+			attributes: {
+				service: variation.name,
+				url: '#',
+			},
+		};
```

Comment:
I think adding a `viewportWidth` in the `example` object would be good as the preview is quite small and scaled down.

A value of `0` results in no scaling which is an option. `150` results in the icon nicely centered but zoomed in and showing some pixelation.